### PR TITLE
(DOCS-4329) High CPU or Memory Troubleshooting

### PR DESCRIPTION
**Pull request by hestonhoffman**
[DOCS-4329](https://datadoghq.atlassian.net/browse/DOCS-4329) 

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

[DOCS-4329]: https://datadoghq.atlassian.net/browse/DOCS-4329?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by urseberry** [comment]
https://datadoghq.atlassian.net/browse/DOCS-4793

File: content/en/agent/troubleshooting/high_memory_usage.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+Several factors can cause high Agent CPU or memory consumption. If you try the steps below and continue to have trouble, [contact Datadog Support for further assistance](#reach-out-to-datadog-support).
+
+Common causes of high CPU or memory consumption:
+
+- An integration is returning thousands of metrics, or is running a large number of check instances. You can see a summary of the running check instances, as well as the number of metrics collected, by running [the `status` CLI command][2] and checking the **Collector** section.
+- The Agent's Python or Go runtime is causing high resource consumption. Enable [Live Processes Monitoring][3] to check if the Agent process is consuming unexpected amounts of memory or CPU. You can also use your operating system's activity manager to check Agent process resource consumption.
+- The Agent is monitoring a large number of processes. This is configured in the [Process Check configuration file][4].
+- The Agent's behavior is triggering Windows anti-malware or antivirus tools, causing high CPU usage.
+- The Agent is forwarding a very large number of log lines or DogstatsD metrics.
```
Comment (cswatt) [inline content/en/agent/troubleshooting/high_memory_usage.md:22]:
```suggestion
- The Agent is forwarding a very large number of log lines or DogStatsD metrics.
```

File: content/en/agent/troubleshooting/high_memory_usage.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  tag: "Agent Troubleshooting"
+  text: "Send an Agent Flare"
+- link: "/agent/troubleshooting/agent_check_status/"
+  tag: "Agent Troubleshooting"
+  text: "Get the Status of an Agent Check"
+---
+
+Several factors can cause high Agent CPU or memory consumption. If you try the steps below and continue to have trouble, [contact Datadog Support for further assistance](#reach-out-to-datadog-support).
+
+Common causes of high CPU or memory consumption:
```
Comment (cswatt) [inline content/en/agent/troubleshooting/high_memory_usage.md:16]: should we make this an H3 or so? might make the page easier to parse at-a-glance
```suggestion
### Common causes of high CPU or memory consumption:
```

File: content/en/agent/troubleshooting/high_memory_usage.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+- An integration is returning thousands of metrics, or is running a large number of check instances. You can see a summary of the running check instances, as well as the number of metrics collected, by running [the `status` CLI command][2] and checking the **Collector** section.
+- The Agent's Python or Go runtime is causing high resource consumption. Enable [Live Processes Monitoring][3] to check if the Agent process is consuming unexpected amounts of memory or CPU. You can also use your operating system's activity manager to check Agent process resource consumption.
+- The Agent is monitoring a large number of processes. This is configured in the [Process Check configuration file][4].
+- The Agent's behavior is triggering Windows anti-malware or antivirus tools, causing high CPU usage.
+- The Agent is forwarding a very large number of log lines or DogstatsD metrics.
+
+Here are some adjustments you can make to your Agent configuration to reduce resource usage:
```
Comment (cswatt) [inline content/en/agent/troubleshooting/high_memory_usage.md:24]:
```suggestion
### Here are some adjustments you can make to your Agent configuration to reduce resource usage:
```

same as above, but i'm not super opinionated about this

**Review by cswatt — CHANGES_REQUESTED** [review]

**Review by cswatt — APPROVED** [review]

**Pull request merged**